apps/src/lab2/views/dialogs/ShareDialog.tsx

```diff
@@ -28,65 +28,108 @@ const CopyToClipboardButton: React.FunctionComponent<{
+        type="primary"
+        color="black"
+        size="m"
+        onClick={handleCopyToClipboard}
       />
-      {i18n.copyLinkToProject()}
-    </button>
+    </div>
   );
 };
 
 /**
  * A new implementation of the project share dialog for Lab2 labs.  Currently only used
  * by Music Lab and Python Lab, and only supports a minimal subset of functionality.
  */
+
 const ShareDialog: React.FunctionComponent<{
+  id?: string;
   shareUrl: string;
   finishUrl?: string;
   projectType: ProjectType;
-}> = ({shareUrl, finishUrl, projectType}) => {
+}> = ({id, shareUrl, finishUrl, projectType}) => {
   const dispatch = useAppDispatch();
 
   useEffect(() => {
-    trackEvent('share', 'share_open_dialog', {value: projectType});
+    trackEvent('congrats', 'congrats_open_dialog', {value: projectType});
   });
 
   const handleClose = useCallback(
     () => dispatch(hideShareDialog()),
     [dispatch]
   );
 
+  const careersUrl =
+    'https://www.amazonfutureengineer.com/careertours/careervideos';
+
   return (
     <FocusLock>
       <div className={moduleStyles.dialogContainer}>
         <div id="share-dialog" className={moduleStyles.shareDialog}>
-          <Typography semanticTag="h1" visualAppearance="heading-lg">
-            {i18n.shareTitle()}
+          <Typography
+            semanticTag="h1"
+            visualAppearance="heading-lg"
+            className={moduleStyles.heading}
+          >
+            {id === 'hoc2024' ? i18n.congratulations() : i18n.shareTitle()}
           </Typography>
-          <div className={moduleStyles.itemsContainer}>
-            <CopyToClipboardButton
-              shareUrl={shareUrl}
-              projectType={projectType}
-            />
-            <div id="share-qrcode-container">
-              <QRCode value={shareUrl + '?qr=true'} size={140} />
+          <div className={moduleStyles.columns}>
+            <div className={moduleStyles.column}>
+              <div className={moduleStyles.share}>
+                <div id="share-qrcode-container">
+                  <QRCode value={shareUrl + '?qr=true'} size={140} />
+                </div>
+                <CopyToClipboardButton
+                  shareUrl={shareUrl}
+                  projectType={projectType}
+                />
+              </div>
+            </div>
+            <div className={moduleStyles.column}>
+              {id === 'hoc2024' ? (
+                <div className={moduleStyles.careers}>
+                  Learn more about careers in technology and music.
+                  <LinkButton
+                    ariaLabel={i18n.learnMore()}
+                    href={careersUrl}
+                    text={i18n.learnMore()}
+                    type="primary"
+                    color="black"
+                    size="m"
+                    target="_blank"
+                  />
+                </div>
+              ) : (
+                <div>Share your project by using these links.</div>
```

Comment:
Does this need to be localized?

Reply:
Once we have a final design, we should localize.  For now it's just a placeholder string to help design discussions, so I think better to not put into the loc pipeline yet.